File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityBreakStatementTest.cpp
#include "ActivityBreakStatementTest.h"
#include <alf/library/BasicInputOutput.hpp>
#include <string>

void BreakStatementTest()
{
  int a = 0;
  while (true)
  {
    a = a + 1;
    alf::library::BasicInputOutput::WriteLine(std::to_string(a));
    if (a > 10)
    {
      break;
    }
  }
  alf::library::BasicInputOutput::WriteLine(std::string("End: ") + std::to_string(a));
}

File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityBreakStatementTest.cs
/// <filename>
///     ActivityBreakStatementTest.cs
/// </filename>
using Alf.Library;
using System;

public class ActivityBreakStatementTest
{
    public static void Execute()
    {
        int a = 0;
        while (true)
        {
            a = a + 1;
            BasicInputOutput.WriteLine(Convert.ToString(a));
            if (a > 10)
            {
                break;
            }
        }
        BasicInputOutput.WriteLine("End: " + Convert.ToString(a));
    }

    public static void Main(string[] args)
    {
        ActivityBreakStatementTest.Execute();
    }
}
